dist/_patterns/01-components/01-page-elements/hero.twig

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero__content" role=”banner”>
+    <h1 class="hero__header">{{ pageTitle }}</h1>
+    <h2 class="hero__subheader">{{ pageSubTitle }}</h2>
+    <p class="hero__text">{{ summary }}</p>
+    <div class="hero__cta">
```

Comment:
I’d prefer a more descriptive class name than hero__cta, perhaps hero__call-to-action?

We should also wrap this in an if statement so that we can hide if necessary when including this pattern.